CosmicReco/fcl/prolog.fcl

```diff
@@ -78,7 +78,7 @@ CosmicReco : { @table::CosmicReco
 
    CosmicTrackFinder : { #split into Loose and tight below for efficieny studies
       module_type                   : CosmicTrackFinder
-      ComboHitCollection     : "makePH"
+      ComboHitCollection     : "makeSH"
```

Comment:
If cosmicreco never uses panel hits I suggest overwriting makePH with RecoNullProducer as is done [here](https://github.com/Mu2e/Offline/blob/dd31c82e4e5c4edeea674b8a894c24acffccd39f/TrkHitReco/fcl/prolog.fcl#L127), then no time will be wasted on calling it.

Reply:
Added dummy producers to prolog, will have to change in Production to implement in reco

Reply:
You could name these instances makePH, makeSTH, then no production change is neeeded